port/common/j9nls.c

```diff
@@ -234,34 +232,30 @@ writeSyslog(struct OMRPortLibrary *portLibrary, uintptr_t flags, const char *for
 		return;
 	}
 
-	/* Attempt to write string to on-stack buffer */
-	COPY_VA_LIST(copyOfArgs, args);
```

Comment:
You will need to bring back all of the COPY_VA_LIST code.  See other comments about this.